README.md

```diff
@@ -9,10 +9,21 @@
 
 # Primaza project
 
-Quarkus Primaza Service box Application - POC
+Quarkus Primaza Service Box Application - POC
+
+Application developers need access to backing services to build and connect workloads.
+Today in Kubernetes, the exposure of secrets for connecting workloads to external services such as REST APIs, databases, event buses, and many more is manual and custom-made.
+Connecting workloads to backing services is always a challenge because each service provider suggests a different way to access their secrets, and each application developer consumes those secrets in a custom way to their workloads.
+Primaza aims to support the discoverability, life cycle management and connectivity of services running in Kubernetes.
 
 Primaza comes from the Greek word πρυμάτσα, which is a line used to tie boats to the dock.
```

Comment:
I think this sentence is a bit out of context in the middle of the previous and the next blocks. Maybe moving it to the end of the section?

Reply:
yes, I aggree. Move to the end of the paragraphe.